### ISSUE TITLE
When using a Skip or Skip If in a suite teardown, an exception is thrown

### ISSUE THREAD

**Issue by david-leroux** [robotframework/robotframework#3901]
Using the following robot:

robot --version
Robot Framework 4.0 (Python 3.6.9 on linux)

I am using a `Skip If    <condition>    Message` in the `Suite Teardown` section of a suite.

This command fails with the following exception.

```
14:38:14  [ ERROR ] Unexpected error: AttributeError: 'SuiteMessage' object has no attribute 'also_teardown_skip_message'
14:38:14  Traceback (most recent call last):
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/utils/application.py", line 83, in _execute
14:38:14      rc = self.main(arguments, **options)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/run.py", line 447, in main
14:38:14      result = suite.run(settings)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/model.py", line 292, in run
14:38:14      self.visit(runner)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/model/testsuite.py", line 195, in visit
14:38:14      visitor.visit_suite(self)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/model/visitor.py", line 87, in visit_suite
14:38:14      suite.suites.visit(self)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/model/itemlist.py", line 77, in visit
14:38:14      item.visit(visitor)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/model/testsuite.py", line 195, in visit
14:38:14      visitor.visit_suite(self)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/model/visitor.py", line 90, in visit_suite
14:38:14      self.end_suite(suite)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/suiterunner.py", line 104, in end_suite
14:38:14      self._suite.message = self._suite_status.message
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/status.py", line 147, in message
14:38:14      return self._my_message()
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/status.py", line 169, in _my_message
14:38:14      return SuiteMessage(self).message
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/status.py", line 232, in message
14:38:14      return self._get_message_after_teardown(message)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/status.py", line 258, in _get_message_after_teardown
14:38:14      return self._format_message_with_teardown_message(message)
14:38:14    File "/usr/local/lib/python3.6/dist-packages/robot/running/status.py", line 270, in _format_message_with_teardown_message
14:38:14      return self.also_teardown_skip_message % (teardown, message)
```
If I comment out line 270 in status.py, as below:
```
266         elif message.startswith('*HTML*'):
267             teardown = html_escape(teardown)
268         if self.failure.teardown:
269             return self.also_teardown_message % (message, teardown)
270         # return self.also_teardown_skip_message % (teardown, message)
271```

The suite passes with all tests and the teardown being skipped

**Comment by pekkaklarck** [comment]
Is suite setup skipped or failed? If yes, this is a duplicate of #3896 that's already fixed. The fix isn't yet released but RF 4.0.1 will be released somewhat soon.

**Comment by pekkaklarck** [comment]
No updates in a week. I assume this is same as #3896 that's already fixed.
